data/json/mutations/mutations.json

```
@@ -630,9 +630,9 @@
     "id": "MARTIAL_ARTS5",
     "name": "Melee Weapon Training",
     "points": 3,
-    "description": "You have practiced fighting with weapons.  You start with your choice of Eskrima, Fencing, Pentjak Silat, Niten Ichi-Ryu, or Sōjutsu.",
+    "description": "You have practiced fighting with weapons.  You start with your choice of Eskrima, Fencing, Pentjak Silat, Niten Ichi-Ryu, Sōjutsu or Medieval Polearm Fighting",
```

```suggestion
    "description": "You have practiced fighting with weapons.  You start with your choice of Eskrima, Fencing, Medieval Polearm Fighting, Niten Ichi-Ryu, Pentjak Silat, or Sōjutsu.",
```
Alphabetized and re-added missing period. There would still have to be a second pass at this once Swordsmanship is merged.